testing/utils/src/lib/e2e-setup.ts

```diff
@@ -1,64 +1,88 @@
-import path from 'node:path';
-import fs from 'node:fs/promises';
+import { join } from 'node:path';
+import { cp, readFile, rm, stat, writeFile } from 'node:fs/promises';
+import * as console from 'node:console';
 
-export function getE2eAppProjectName(): string | undefined {
-  const e2eProjectName = process.env['NX_TASK_TARGET_PROJECT'] ?? '';
+export function getE2eAppProjectName(): string {
+  const e2eProjectName = process.env['NX_TASK_TARGET_PROJECT'];
   if (e2eProjectName == null) {
     console.warn('NX_TASK_TARGET_PROJECT is not set.');
   }
-  return e2eProjectName ? `${e2eProjectName}-app` : undefined;
+  return e2eProjectName ? `${e2eProjectName}-app` : 'test-e2e-app';
 }
 
-export async function setupE2eApp(
-  fixtureProjectName: string,
-  e2eFixtures?: string
-) {
-  const targetProjectName =
-    getE2eAppProjectName() ?? fixtureProjectName + '-e2e-app';
-  const fixture = path.join(
-    __dirname,
-    `../../../e2e/fixtures/${fixtureProjectName}`
-  );
-  const target = path.join(
-    __dirname,
-    `../../../e2e/__test__/${targetProjectName}`
-  );
+export async function setupE2eApp({
```

Comment:
This function maintains the logic that runs pre-e2e and is responsible to setup a project.